<!DOCTYPE html>
<html lang="en">

<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-112769090-2"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-112769090-2');
</script>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <meta name="description" content="">
  <meta name="author" content="">
  <meta name= "google-site-verification" content ="google5b2cadf776d0cfbe.html">

  <title>Michelle Jonika</title>

  <!-- Bootstrap core CSS -->
  <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

  <!-- Custom fonts for this template -->
  <link href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css?family=Muli:400,400i,800,800i" rel="stylesheet">
  <link href="vendor/fontawesome-free/css/all.min.css" rel="stylesheet">
  <link rel="stylesheet" href="css/academicons.min.css"/>

  <!-- Custom styles for this template -->
  <link href="css/resume.min.css" rel="stylesheet">
  <!--Hero image test for the website-->
  <link href="css/hero_image.css" rel="stylesheet">

</head>

<body id="page-top">

  <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
    <a class="navbar-brand js-scroll-trigger" href="#page-top">
      <span class="d-block d-lg-none">Michelle Jonika</span>
      <span class="d-none d-lg-block">
        <img class="img-fluid img-profile square mx-auto mb-2" src="img/profile.jpg" alt="">
      </span>
    </a>
    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>
    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav">
        <li class="nav-item">
          <a class="nav-link js-scroll-trigger" href="#about">Home</a>
        </li>
        <li class="nav-item">
          <a class="nav-link js-scroll-trigger" href="#experience">Education &amp; Experience</a>
        </li>
        <li class="nav-item">
          <a class="nav-link js-scroll-trigger" href="#education">Research</a>
        </li>
        <li class="nav-item">
          <a class="nav-link js-scroll-trigger" href="#skills">Publications</a>
        </li>
        <li class="nav-item">
          <a class="nav-link js-scroll-trigger" href="#interests">Skills</a>
        </li>
        <li class="nav-item">
          <a class="nav-link js-scroll-trigger" href="#awards">Awards</a>
        </li>
        <li class="nav-item">
          <a class="nav-link js-scroll-trigger" href="#news">News</a>
        </li>
      </ul>
    </div>
  </nav>
<div class="hero-image", id="about">
  <div class="hero-text">

  </div>
</div>

  <div class="container-fluid p-0">

    <section class="resume-section p-3 p-lg-5 d-flex align-items-center" id="about">
      <div class="w-100">
        <h1 class="mb-0">Michelle
          <span class="text-primary">Jonika</span>
        </h1>
        <div class="subheading mb-5">Genetics and Genomics PhD Student · Blackmon Lab · Department of Biology · Texas A&amp;M University <br><a href="mailto:name@email.com">Michelle19@tamu.edu</a><br><br>
        <a href="Jonika-CV.pdf">Current CV</a> |
        <a href="Jonika-Resume.pdf">Current Resume</a>
        </div>
        <p class="lead mb-5">Evolutionary biologist using big data, bioinformatics, machine learning, and genomics to answer broad evolutionary questions. Check out some of my research projects below! </p>
        <div class="social-icons">
          <a href="https://www.linkedin.com/in/michelle-jonika-7b2459138/">
            <i class="fab fa-linkedin-in"></i>
          </a>
          <a href="https://github.com/MichelleJonika">
            <i class="fab fa-github"></i>
          </a>
          <a href="https://twitter.com/michellejonika">
            <i class="fab fa-twitter"></i>
          </a>
          <a href="https://www.researchgate.net/profile/Michelle_Jonika">
              <i class="fab fa-researchgate"></i>
          </a>
        </div>
      </div>
    </section>

    <hr class="m-0">

    <section class="resume-section p-3 p-lg-5 d-flex justify-content-center" id="experience">
      <div class="w-100">
        <h2 class="mb-5">Education &amp; Experience</h2>

        <div class="resume-item d-flex flex-column flex-md-row justify-content-between mb-5">
          <div class="resume-content">
            <h3 class="mb-0">PhD in Genetics</h3> <p> Graduate Certificate in Business </p>
            <a href="https://genetics.tamu.edu/"><img src="img/logo.jpg" height = "45" width="45" alt="" style=float:left></a>
            <div class="subheading mb-3"> &ensp;Texas A&amp;M University, College Station, TX
            </div>
              <p>
                  The content of genomes can be categorized into different sequence classes, into autosomes and sex chromosomes, coding and noncoding, repetitive and non-repetitive, to name but a few. Each of these classes of the genome have unique mechanisms that govern their evolution. In my dissertation work, I am studying the evolution of the genome on these three scales. <br>
                  Advised by: <a href="http://coleoguy.github.io/">Heath Blackmon</a>
              </p>
          </div>
          <div class="resume-date text-md-right">
            <span class="text-primary">August 2018 - Present</span>
          </div>
        </div>
          
        <div class="resume-item d-flex flex-column flex-md-row justify-content-between mb-5">
          <div class="resume-content">
            <h3 class="mb-0">Bayer Crop Science</h3> <p> Data Science Intern | Genomics Discovery and Application Intern </p>
            <a href="https://www.cropscience.bayer.com/"><img src="img/bayer_logo.svg" height = "45" width="45" alt="" style=float:left></a>
            <div class="subheading mb-3"> &ensp;St.Louis, Missouri (Virtual)
            </div>
              <ul style="list-style-type:disc">
                  <li>Identifying historic data to test for epistasis</li>
                  <li>Developing a statistical testing framework to identify interactions between introgressed loci</li>
                  <li>Three-month, full-time position exposure inn an industry setting</li>
                  <li>Establishig multi-disciplinary connections with teams with expertise in data science, genommics, and precision breeding</li>
              </ul>
          </div>
          <div class="resume-date text-md-right">
            <span class="text-primary">May 2021 - August 2021</span>
          </div>
        </div>

        <div class="resume-item d-flex flex-column flex-md-row justify-content-between mb-5">
          <div class="resume-content">
            <h3 class="mb-0">Bachelor of Science in Forensic and Investigative Science </h3> <p>Minor: Genetics</p>
            <a href="https://entomology.tamu.edu/forensic-investigative-sciences/"><img src="img/logo.jpg" height = "45" width="45" alt="" style=float:left></a>
            <div class="subheading mb-3"> &ensp;Texas A&amp;M University, College Station, TX</div>
            <p>Undergraduate Thesis: Genes as Markers of Sex for Forensic Entomology — Identifying genes that can be used for a transcriptional approach to sex identification in blow fly species of forensic importance: Lucilia sericata (Diptera: Calliphoridae) (Meigen), Cochliomyia macellaria (Diptera: Calliphoridae) (Fabricius), and Chrysomya rufifacies (Diptera: Calliphoridae) (Macquart). <br>
            Advised by: <a href="https://taronelab.tamu.edu/lab-members/michelle-jonika/">Aaron M. Tarone</a></p>
          </div>
          <div class="resume-date text-md-right">
            <span class="text-primary">August 2014-May 2018</span>
          </div>
        </div>
        </div>
    </section>

    <hr class="m-0">
    <section class="resume-section p-3 p-lg-5 d-flex align-items-center" id="education">
      <div class="w-100">
        <h2 class="mb-5">Research</h2>

        <div class="resume-item d-flex flex-column flex-md-row justify-content-between mb-5">
          <div class="resume-content">
            <h3 class="mb-0">Microsatellite Evolution</h3>
            <p>Microsatellites are short, repetitive DNA sequences that can rapidly expand and contract due to slippage during DNA replication. Despite their impacts on transcription, genome structure, and disease, relatively little is known about the evolutionary dynamics of these short sequences across long evolutionary periods. To address this gap in our knowledge, we performed comparative analyses of 304 available insect genomes. We investigated the impact of sequence assembly methods and assembly quality on the inference of microsatellite content, and we explored the influence of chromosome type and number on the tempo and mode of microsatellite evolution across one of the most speciose clades on the planet. Diploid chromosome number had no impact on the rate of microsatellite evolution or the amount of microsatellite content in genomes. We found that centromere type (holocentric or monocentric) is not associated with a difference in the amount of microsatellite content; however, in those species with monocentric chromosomes, microsatellite content tends to evolve faster than in species with holocentric chromosomes.</p>
          </div>
        </div>
          
          <div class="resume-item d-flex flex-column flex-md-row justify-content-between mb-5">
          <div class="resume-content">
            <h3 class="mb-0">Holocentric Chromosomes</h3>
            <p>Despite the fundamental role of centromeres two different types are observed across plants and animals. Monocentric chromosomes possess a single region that function as the centromere while in holocentric chromosomes centromere activity is spread across the entire chromosome. Proper segregation may fail in species with monocentric chromosomes after a fusion or fission, which may lead to chromosomes with no centromere or multiple centromeres. In contrast, species with holocentric chromosomes should still be able to safely segregate chromosomes after fusion or fission. This along with the observation of high chromosome number in some holocentric clades has led to the hypothesis that holocentricity leads to higher rates of chromosome number evolution. To test for differences in rates of chromosome number evolution between these systems, we analyzed data from 4,393 species of insects in a phylogenetic framework. We found that insect orders exhibit striking differences in rates of fissions, fusions, and polyploidy. However, across all insects we found no evidence that holocentric clades have higher rates of fissions, fusions, or polyploidy than monocentric clades. Our results suggest that holocentricity alone does not lead to higher rates of chromosome number changes. Instead, we suggest that other co-evolving traits must explain striking differences between clades.</p>
          </div>
        </div>

        <div class="resume-item d-flex flex-column flex-md-row justify-content-between">
          <div class="resume-content">
            <h3 class="mb-0">Mammalian Pseudoautosomal Region</h3>
            <p>The classic model of sex chromosome evolution begins with a pair of homomorphic sex chromosomes and leads to eventual decay of the Y chromosome in regions of suppressed recombination. Despite degradation of the Y chromosome, a pseudo-autosomal region (PAR) persists in most species. The PAR is a region of homology between sex chromosomes that maintains recombination and is essential for segregation of sex chromosomes during meiosis. What happens to sex chromosomes once the PAR becomes very small? The fragile Y hypothesis predicts that as PAR sizes become smaller, the risk of aneuploidy increases, increasing the probability of Y chromosome loss, achiasmatic meiosis, or rejuvenation through translocation or fusion of autosomal material. While these outcomes are perceived as rare due to their absence in most mammals, each of them is quite common across the tree of life and all have occurred in mammals. The unique nature of sex chromosomes has made them a focus of evolutionary biology since their discovery more than a century ago, but PAR size has been estimated in just 9 species of mammals. For this reason, many questions have remained unanswered due to limitations of available data and analysis tools. <br><br>
            Current methods to estimate PAR sizes are time consuming and costly. For this reason, I developed an approach to train and deploy a convolutional neural network (CNN) to accurately predict PAR size. I am currently using my developed pipeline to generate training datasets and hope to train and deploy the CNN in the next few months. The trained CNN will allow for estimation of PAR size for hundreds of species both with publicly available genomic sequences and through de-novo sequencing of DNA available from collaborators. Additionally, this tool will be made broadly available to the community as a publicly available genome annotation tool to improve all future genome assemblies. With vastly increased numbers of PAR size estimates, we can for the first time begin to understand the dynamics of this genomic trait and determine whether achiasmatic meiosis, fusions of autosomes to sex chromosomes, and Y loss are associated with small PAR sizes.</p>
          </div>
        </div>

      </div>
    </section>

    <hr class="m-0">

    <section class="resume-section p-3 p-lg-5 d-flex justify-content-center" id="skills">
      <div class="w-100">
        <h2 class="mb-5">Publications</h2>
          <h3 class="mb-0">Check out my <a href="https://scholar.google.com/citations?user=1-NTzLkAAAAJ&hl=en">Google Scholar</a> page</h3>

        <div class="resume-item d-flex flex-column flex-md-row justify-content-between">
          <div class="resume-content">
            <h3 class="mb-0">2021</h3>
            <p></p>
            <p>M.L. Pimsler, C.E. Hjelmen, <b>M.M. Jonika</b>, A. Sharma, S. Fu, M. Bala, S.H. Sze, J.K. Tomberlin, A.M. Tarone. 2021. Sexual Dimorphism in Growth Rate and Gene Expression Throughout Immature Development in Wild Type Chrysomya rufifacies (Diptera: Calliphoridae). Frontiers in Ecology and Evolution 9:368 <a href="2021.pdf">[PDF]</a></p>
          </div>
         </div>
          
          <div class="resume-item d-flex flex-column flex-md-row justify-content-between">
          <div class="resume-content">
            <h3 class="mb-0">2020</h3>
            <p></p>
            <p>S. Ruckman* (Co-first author), <b>M.M. Jonika* (Co-first author)</b>, C. Casola, H. Blackmon. 2020. Chromosome Number Evolves at Equal Rates in Holocentric and Monocentric Clades. PLOS Genetics 16(10):e1009076 <a href="2020c.pdf">[PDF]</a></p>
            <p><b>M.M Jonika</b>, J. Lo, H. Blackmon. 2020. Mode and Tempo of Microsatellite Evolution across 300 Million Years of Insect Evolution. Genes 11, 945 <a href="2020b.pdf">[PDF]</a></p>
            <p><b>M.M. Jonika</b>, C.E. Hjelmen, A.M. Faris, A.M. Tarone. 2020. An Evaluation of Differentially Spliced Genes As Markers of Sex for Forensic Entomology. Journal of Forensic Science. 65:1579-1587 <a href="2020a.pdf">[PDF]</a></p>
          </div>
         </div>
          
          <div class="resume-item d-flex flex-column flex-md-row justify-content-between">
          <div class="resume-content">
            <h3 class="mb-0">2019</h3>
            <p>J. Lo, <b>M.M. Jonika</b>, H. Blackmon. 2019. micRocounter: Microsatellite Characterization in Genome Assemblies. G3: Genes | Genomes | Genetics 9(10):3101-3104 <a href="2019d.pdf">[PDF]</a></p>
            <p>R.D. Perkins, J.R. Gamboa, <b>M.M. Jonika</b>, J. Lo, A. Shum, R.H Adams, H. Blackmon. 2019. A Database of Amphibian Karyotypes. Chromosome Research 27:313-319 <a href="2019c.pdf">[PDF]</a></p>
          </div>
         </div>


        <div class="resume-item d-flex flex-column flex-md-row justify-content-between">
          <div class="resume-content">
            <h3 class="mb-0">2018</h3>
            <p>A.B. Blake, B.C. Guard, J.B. Honneffer, <b>M.M. Jonika</b>, J. Chaitman, J.A. Lidbury, J.M. Steiner and J.S. Suchodolski. 2018. Altered Fecal Fatty Acid, Sterol and Bile Acid Metabolism in Dogs with Acute Diarrhea. Journal of Veterinary Internal Medicine 32:2248.<a href="2018b.pdf">[PDF]</a></p>
          </div>
         </div>
      </div>

    </section>


    <hr class="m-0">

    <section class="resume-section p-3 p-lg-5 d-flex align-items-center" id="interests">
      <div class="w-100">
        <h2 class="mb-5">Skills</h2>
        <ul class="fa-ul mb-0">
          <li>
            <i class="fa-li fa fa-check"></i>
            <b>Genomics:</b> SRA, Trimmomatic, bwa, samtools, bamtools, GATK, PLINK, BUSCO, rrBLUP, GEMMA </li>
          <li>
            <i class="fa-li fa fa-check"></i>
            <b>Genetics:</b> Genome structure evolution, Sex Chromosome evolution, Morphometrics, Genomics analysis, Genome Assembly </li>
          <li>
            <i class="fa-li fa fa-check"></i>
            <b>Molecular Biology:</b> Primer optimization, gDNA/DNA extraction, RNA extraction, PCR, qPCR, Gel visualization/imaging</li>
          <li>
            <i class="fa-li fa fa-check"></i>
            <b>Programming:</b> R, Unix/Linux, Git, tidyverse, conda, SQL, Python, LaTeX, HTML/CSS </li>
          <li>
            <i class="fa-li fa fa-check"></i>
            <b>Data Science:</b> Large dataset management (>20 Gb), Bayesian statistics, Phylogenetics, Simulations, Data Visualization (ggplot2)</li>
          <li>
            <i class="fa-li fa fa-check"></i>
            <b>Soft Skills:</b> Project management, Public Speaking, Leadership, Multi-disciplinary collaboration, Adaptive problem solving</li>
        </ul>
      </div>
    </section>

    <hr class="m-0">

    <section class="resume-section p-3 p-lg-5 d-flex align-items-center" id="awards">
      <div class="w-100">
        <h2 class="mb-5">Awards &amp; Certifications</h2>
        <ul class="fa-ul mb-0">
          <li>
              <i class="fa-li fa fa-trophy text-warning"></i>
              Oustanding PhD Student Poster Presentation, Texas Genetics Society, Virtual</li>
          <li>
              <i class="fa-li fa fa-trophy text-warning"></i>
              Oustanding PhD Student Oral Presentation, North American Forensic Entomology Association Annual Meeting, Virtual</li>
          <li>
              <i class="fa-li fa fa-trophy text-warning"></i>
              Genetics Graduate Student Association Oral Presentation Award, Texas A&amp;M University</li>
          <li>
              <i class="fa-li fa fa-trophy text-warning"></i>
              Texas Ecolab Fund Grant, "Evolution of Sex Chromosomes in Tiger Beetle Species of the Southern U.S." </li>
          <li>
              <i class="fa-li fa fa-trophy text-warning"></i>
              <a href="https://www.facebook.com/TAMUGENE/posts/696840734074527">1st Place Genetics Science Art Competition, Texas A&amp;M University</a></li>
          <li>
            <i class="fa-li fa fa-trophy text-warning"></i>
              <a href="https://www.facebook.com/TAMUGENE/posts/663573384067929">Genetics Graduate Student Travel Award, Texas A&amp;M University</a></li>
          <li>
        </ul>
      </div>
    </section>

    <hr class="m-0">

    <section class="resume-section p-3 p-lg-5 d-flex align-items-center" id="news">
      <div class="w-100">
        <h2 class="mb-5">News</h2>
        <ul class="fa-ul mb-0">
          <li>
              <a class="twitter-timeline" data-width="600" data-height="350" data-theme="light" href="https://twitter.com/michellejonika?ref_src=twsrc%5Etfw">Tweets by michellejonika</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
            </li>
        </ul>
      </div>
    </section>

  </div>

  <!-- Bootstrap core JavaScript -->
  <script src="vendor/jquery/jquery.min.js"></script>
  <script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

  <!-- Plugin JavaScript -->
  <script src="vendor/jquery-easing/jquery.easing.min.js"></script>

  <!-- Custom scripts for this template -->
  <script src="js/resume.min.js"></script>

</body>

</html>
